multipaz/src/commonMain/kotlin/org/multipaz/rpc/handler/RpcNonceAndSession.kt

```diff
@@ -0,0 +1,183 @@
+package org.multipaz.rpc.handler
+
+import kotlinx.coroutines.sync.Mutex
+import kotlinx.coroutines.sync.withLock
+import kotlinx.io.bytestring.ByteString
+import org.multipaz.cbor.Bstr
+import org.multipaz.crypto.Algorithm
+import org.multipaz.crypto.Crypto
+import org.multipaz.device.AssertionRpcAuth
+import org.multipaz.rpc.backend.BackendEnvironment
+import org.multipaz.storage.Storage
+import org.multipaz.storage.StorageTable
+import org.multipaz.storage.StorageTableSpec
+import org.multipaz.util.Logger
+import org.multipaz.util.fromBase64Url
+import org.multipaz.util.toBase64Url
+import kotlin.concurrent.Volatile
+import kotlin.experimental.xor
+import kotlin.math.min
+import kotlin.random.Random
+import kotlin.time.Clock
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.minutes
+import kotlin.time.Instant
+
+class RpcNonceAndSession(
+    val nextNonce: ByteString,
+    val sessionId: String
+) {
+    companion object {
+        const val TAG = "RpcNonceAndSession"
```

Comment:
private... also many other places... it's important to fix this b/c otherwise it ends up accidentally getting importing when people copy-paste to another file w/o TAG defined.